tests/rbd_mirror/test_rbd_mirror_image_features.py

```diff
@@ -0,0 +1,102 @@
+import time
+
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verification of change in properties of image in Primary Pool
+    gets reflected on remote site upon modification.
+
+    Args:
+        **kw:
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+
+    Test case covered -
+    CEPH-9520 - When a both Pool and image based mirroring is established,
+    verify if change of any image features in primary site should reflect
+    on remote site upon modification.
+
+
+    Pre-requisites :
+    1. Two Clusters must be up and running to create pool
+    2. We need atleast one client node with ceph-common package,
+       conf and keyring files on each node.
+
+    Test Case Flow:
+    1. Create a pool on both clusters.
+    2. Configure mirroring (peer bootstrap) between two clusters.
+    3. Enable pool mode journal based mirroring on the pool respectively.
+    4. Verify if change of any image features in primary site gets mirrored in secondary site.
+    5. Enable image mode snapshot based mirroring on the pool respectively.
+    6. Verify if change of any image features in primary site gets mirrored in secondary site.
+    """
+
+    try:
+        log.info("Running RBD mirroring test case for image features reflection")
+        config = kw.get("config")
+        mirror1, mirror2 = [
```

Comment:
Try using the rbd_mirror_config method from mirror_utils, so we can include ec pool inclusion as well as part of test case automation itself.
We need not work on ec pool inclusion separately.

Reply:
Done